mavros/include/mavros/frame_tf.h

```diff
@@ -186,6 +188,26 @@ inline T transform_orientation_baselink_aircraft(const T &in) {
 	return detail::transform_orientation(in, StaticTF::BASELINK_TO_AIRCRAFT);
 }
 
+/**
+ * @brief Transform from attitude represented WRT aircraft frame to
+ *		  attitude represented WRT base_link frame, treating aircraft frame 
+ *		  as if it were fixed frame as opposed to the function above.
```

Comment:
What do you mean with "fixed" frame?

Reply:
Do you mean inertial frame instead?

Reply:
Sorry, I meant "absolute" from http://www.euclideanspace.com/maths/algebra/realNormedAlgebra/quaternions/transforms/

"absolute frame of reference"
changing

Reply:
Right makes sense. Can you please then change the fixed/FIXED refetences to absolute/ABSOLUTE?